.gitignore

```diff
@@ -5,6 +5,7 @@ __pycache__
 
 /data/*
 /experiments/*
-/notebooks/*
```

Comment:
I'm not sure if this is a good idea. Personally, I create a lot of notebooks for quick debugging, data exploration, etc., and I do not want them to show up in git. So unless it really breaks your workflow, I would prefer to keep the notebooks folder in .gitignore. If you really need git to track some notebook, you can use `git add --force`.